ERCS/erc-7606.md

```diff
@@ -0,0 +1,75 @@
+---
+eip: 7606
+title: Transaction risk monitoring
+description: Establishing a framework for monitoring and regulating high-risk
```

Comment:
Your description isn't a complete sentence, nor does it really enough information that wasn't already in the title.